interfaces/console/src/app/console/home/page.tsx

```diff
@@ -148,7 +112,7 @@ export default function Page() {
             subtitle1={`${networkStats.sales}`}
             Icon={SalesIcon}
             options={MONTH_FILTER}
-            loading={statLoading}
+            loading={false}
```

Comment:
Why are we setting loading to false.

Reply:
I had to remove the Home page stats for now, because those KPI's data is not coming from prometheus, These stats are based on our backend data. and that behaviour is missing.

I have opened the issue, Thanks for mentioning
https://github.com/ukama/ukama/issues/927